client/app/index.html

```diff
@@ -195,6 +195,8 @@
     <script src="components/fusty-flow.js/src/fusty-flow.js"></script>
     <script src="components/fusty-flow.js/src/fusty-flow-factory.js"></script>
     <script src="components/ng-flow/dist/ng-flow.min.js"></script>
+    <script src="components/zxcvbn/dist/zxcvbn.js"></script>
+    <script src="components/angular-zxcvbn/dist/angular-zxcvbn.js"></script>
```

Comment:
@evilaliv3  neither `components/zxcvbn/dist/zxcvbn.js` or `components/angular-zxcvbn/dist/angular-zxcvbn.js` are included as files in this PR. 

In development mode running this does not work